lib/migrator/migrationSteps.js

```diff
@@ -120,46 +120,78 @@ module.exports.removeOldPatternIncludesFromSassFile = async (patterns, sassFile)
   })
   const succeeded = await writeFileLinesToFile(filePath, updatedContent)
   await reporter(succeeded)
+  return succeeded
 }
 
 module.exports.deleteUnusedFiles = async (filesToDelete) => {
   const reporter = await addReporter('Deleted files that are no longer needed')
-  const results = await Promise.all(filesToDelete.map(file =>
-    deleteFile(path.join(projectDir, file))))
+  const results = await Promise.all(filesToDelete.map(async file => {
+    const filePath = path.join(projectDir, file)
+    if (!await fse.pathExistsSync(filePath)) {
```

Comment:
We can either use the async version of remove the await, no need to await sync function

```suggestion
    if (!await fse.pathExists(filePath)) {
```